core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -186,20 +187,23 @@ static paddr_t core_mmu_get_ul1_ttb_pa(void)
 	return pa;
 }
 
-static void *core_mmu_alloc_l2(struct tee_mmap_region *mm)
+static void *core_mmu_alloc_l2(size_t size)
```

Comment:
@jenswi-linaro   But this P-R is a base for dynamic SHM mapping,  if we don't do it here, we need to do it in the following patches, and i think the pgt_cache is  used for user TAs mapping, and here we are handling the kernel mapping, do i miss anything here or misunderstanding?